src/Client.js

```diff
@@ -52,4 +52,12 @@ export default class Client {
       { email },
     )).data;
   }
+
+  async resetPassword(email, token, password) {
+    const { protocol, hostname, port } = this.options;
+    return (await axios.post(
```

Comment:
The same about a `response` variable to separate from putting all together on return statement.